ReactQt/runtime/src/componentmanagers/textmanager.cpp

```diff
@@ -130,6 +132,36 @@ QVariant TextManager::nestedPropertyValue(QQuickItem* item, const QString& prope
     return item->property(propertyName.toStdString().c_str());
 }
 
+void TextManager::click(QQuickItem* textItem) {
+    qDebug() << "### click received";   
+
+    QPointF lp(0,0);
+    QPointF local(0,0);
+
+    QVariantMap e;
+    AttachedProperties* ap = AttachedProperties::get(textItem, false);
+    e.insert("target", ap->tag());
```

Comment:
Filling `QVariantMap` looks very similar in `RootView::makeReactTouchEvent`. It would be nice to move common code to single function, for example `utilities::createTouchArgs(tag, lp, local)`

Reply:
Yes, I've just added a comment to the function that it's a simplified version of makeReactTouchEvent(). Will refactor.

Reply:
@siphiuel, please also remove `console.log` and `qDebug()` calls before merging. And changes in `react-native-cli/index.js` :)